.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+# On every pull request, but only on push to master
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

Comment:
Should this be `runs-on: ${{ matrix.os }}` and under matrix `os: [ ubuntu-latest, windows-latest, macos-latest ]`?

Reply:
I kept it single-os for now to make this a direct port of the previous travis.yml file. Maybe merge this and then have a new pr to add multiple OS-es?